dev/manual_tests/lib/actions.dart

```diff
@@ -1,13 +1,19 @@
 // Copyright 2019 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'dart:io';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
 void main() {
+  if (Platform.isMacOS) {
+    // See https://github.com/flutter/flutter/wiki/Desktop-shells#target-platform-override
```

Comment:
We should probably annotate this (throughout the PR) with the bug number to make it easier to find and remove places we've done this later.

Reply:
Sure. Added a TODO referencing flutter/flutter#31366, but left the link to the shell Wiki too.